config/session.php

```diff
@@ -150,4 +150,19 @@
 
     'secure' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Same-Site Cookies
+    |--------------------------------------------------------------------------
+    |
+    | This option determines how your cookies behave when cross-site requests
+    | take place, and can be used to mitigate CSRF attacks. By default, we
+    | do not enable this as other CSRF protection services are in place.
+    |
+    | Supported: "lax", "strict"
+    |
+    */
+
+    'same_site' => null,
```

Comment:
Could you provide more information on what exactly each of those three options do?

Reply:
The same-site attribute can take one of two values: ‘strict’ or ‘lax’. In strict mode, same-site cookies will be withheld for any kind of cross-site usage. This includes all inbound links from external sites to the application. Visitors clicking on such a link will initially be treated as ‘not being logged in’ whether or not they have an active session with the site.

The lax mode caters to applications which are incompatible with these restrictions. In this mode, same-site cookies will be withheld on cross-domain subrequests (e.g. images or frames), but will be sent whenever a user navigates safely from an external site, for example by following a link.

Reply:
Could you provide a summarized version of that information directly in the doc block for the config option itself?

Reply:
Done, see https://github.com/octobercms/october/pull/3688/commits/8775b8bffaccb237e18fcf4bfa92b17ea4cc458b

Reply:
If we are moving this to the cookies.php we should move the other cookie configurations also to the cookies.php, like:
- secure (HTTPS Only Cookies)
- domain (Session Cookie Domain)
- path (Session Cookie Path)
- cookie (Session Cookie Name)

I think for this PR it's better to keep it in the session.php, we can always move it to the cookie.php (with the other cookie configuration options).

Reply:
Makes sense to me